### ISSUE TITLE
Trying to set up COPR build on release (tagging) time

### ISSUE THREAD

**Issue by martinpitt** [packit/packit-service#1475]
That annoying guy with a support request again... :bowing_man: 

We have used our [cockpituous](https://github.com/cockpit-project/cockpituous/) scrips to automate our cockpit releases for a long time. But we are interested in moving that to packit as much as possible.  One interesting step would be the upload to our [official COPR](https://copr.fedorainfracloud.org/coprs/g/cockpit/cockpit-preview/), and the [packit docs](https://packit.dev/docs/configuration/#copr_build) seem to indicate that this is supported.

So I tested this on my [cockpit-podman fork](https://github.com/martinpitt/cockpit-podman): I [replaced the cockpituous copr release with packit](https://github.com/martinpitt/cockpit-podman/commit/55000eba13644b797685e76ab3efd99cfce51ca8), and added a [second test commit](https://github.com/martinpitt/cockpit-podman/commit/750bd2322f6ce846c0c6d7d24d99c3fbb3076ed1) which moves the COPR to [my test one](https://copr.fedorainfracloud.org/coprs/martinpitt/test-fixes) and disable release steps which don't work on my fork (unrelated).

Then I tagged 46.1 which was [properly released](https://github.com/martinpitt/cockpit-podman/releases/tag/46.1). But there is still [no build](https://copr.fedorainfracloud.org/coprs/martinpitt/test-fixes/builds/) attempt in my COPR. That by itself is understandable, as I didn't get or accept any "builder" privilege request yet, as the documentation says. There [is no GitHub issue](https://github.com/martinpitt/cockpit-podman/issues) (similar to the ones which packit sends for failed koji/bodhi builds), nor did I get any email about it.

How is that supposed to work? Did I mess up for forget something? Or is this not working yet?

Thank you!

**Comment by martinpitt** [comment]
CC @allisonkarlitskaya , we were recently talking about this and you were interested. This would get rid of a secret, and most importantly, it's the secret which is most annoying, as the COPR tokens need to be refreshed every 6 months.

**Comment by martinpitt** [comment]
FTR, there is nothing pending on https://copr.fedorainfracloud.org/coprs/martinpitt/test-fixes/permissions/ either (just in case some email got lost)

**Comment by mfocko** [comment]
Hi, I see it in Sentry, we got a smallish bug after Copr changed the exception they raise for missing privileges, it's been fixed by #1472. For now I'm going to request the builder permissions manually.

In case there is anything else, feel free to reopen the issue or create a new one.

**Comment by mfocko** [comment]
The fix will be deployed Tuesday morning and I have requested the builder permissions.

**Comment by martinpitt** [comment]
Great, thanks @mfocko! I got an email, and approved the permission request. Is there some way to retrigger the packit copr release? or will it auto-retry?

For this test it's easy enough to just delete and re-push the release/tag, but I don't think I can pre-create the permission for our cockpit projects.

**Comment by mfocko** [comment]
I was going to point you to the checks on the commit tied to the tag, but I cannot see any „Retry“ button there, so I'd go with delete and re-push. I'll open a new issue for that or have a look at it.

**Comment by martinpitt** [comment]
Progress! I [recreated the release](https://github.com/martinpitt/cockpit-podman/releases/tag/46.1), and got an automatic [packit srpm build](https://dashboard.packit.dev/results/srpm-builds/65134) with the [corresponding COPR page](https://copr.fedorainfracloud.org/coprs/martinpitt/test-fixes/build/4338595/). 

Now the problem is that the version number is "46.1-1.20220428090917014096.750bd2322f6ce846c0c6d7d24d99c3fbb3076ed1.0.g750bd23.fc35 " instead of the expected "46.1-1.fc35". As this issue is closed, should I file a new one or is that known?

Ideally it would use the exact tarball artifact from the GitHub release page, but rebuilding it is certainly good enough for a COPR (as long as people don't try to upload it to Fedora/RHEL). I haven't checked if it actually does, but even if not that's a compromise I'm willing to do in exchange for losing our custom copr token and mechanism :grin: 

Thanks!

**Comment by mfocko** [comment]
> Now the problem is that the version number is "46.1-1.20220428090917014096.750bd2322f6ce846c0c6d7d24d99c3fbb3076ed1.0.g750bd23.fc35 " instead of the expected "46.1-1.fc35". As this issue is closed, should I file a new one or is that known?

I'll be bumping priority of packit/packit#1181

**Comment by martinpitt** [comment]
Subscribed, thanks @mfocko ! So I'll try this again once that is fixed.

**Comment by martinpitt** [comment]
To conclude this, this is now fixed. I just did a new [47.1 test release](https://github.com/martinpitt/cockpit-podman/releases/tag/47.1) and the [copr build](https://copr.fedorainfracloud.org/coprs/martinpitt/test-fixes/build/4359092/) is now perfect! :+1:
